### PR TITLE
Rebuild with noarch

### LINKED ISSUE #4: All the engarde 0.4.0 builds seem to have failed
Hey @conda-forge/engarde folks, it looks like the CIs for `engarde 0.4.0` have all failed. Could you please restart them? Also, you might be able to do a re-rendering using `noarch: python`.

### PR DESCRIPTION
**Pull request by TomAugspurger**
Closes #4

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,18 +12,19 @@ source:
...
+  noarch: python
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
     - python
     - setuptools
-    - numpy x.x
+    - numpy
     - pandas
```
Comment (jakirkham) [inline recipe/meta.yaml]: Are these still needed at build time?

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender.

File: recipe/meta.yaml (hunk 1 above)
Comment (TomAugspurger) [reply recipe/meta.yaml]: Probably not. Engarde is pure-python, and doesn't import NumPy during setup.py, so should it be run-only? Same with pandas?

Comment (jakirkham) [reply recipe/meta.yaml]: Yep, both can be dropped then.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,18 +12,19 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
+  noarch: python
```
Comment (jakirkham) [inline recipe/meta.yaml:16]: Just as FYI, re-rendering is needed when switching to or from `noarch`. Though now we can ask a bot to do the re-rendering for us. 😄

**Pull request merged**